src/services/aws_macros.ts

```diff
@@ -59,9 +67,26 @@ export class AWS {
   secretsClient: SecretsManager;
   cloudfrontClient: CloudFront;
   memoryDBClient: MemoryDB;
+  slowRetryStrategy: StandardRetryStrategy;
```

Comment:
nit: maybe be more specific here like `apiGatewayRetryStrategy`

Reply:
Actually i wanted to make it generic... we can use it for apiGateway but maybe we can reuse in the future for other apis that are slow?